src/community/contribution-criteria/index.md.njk

```diff
@@ -107,3 +107,19 @@ The working group reviews the implementation to make sure it is usable, consiste
     ]
   ]
 }) }}
+
+
+## Developing a community resource
+
+Before linking from the Design System to a [community resource](/community/resources) the Design System team will review it to make sure that:
+
+- it has a clear owner
+- it cannot be mistaken for an official resource
+- it has documentation explaining how to use it, contribute to and get support for it
+- support tickets are answered within 1 month
+- design resources are no more than 3 months behind the latest release of the GOV.UK Design System 
+- production resources are no more than 1 month behind the latest release of GOV.UK Frontend
+- design resources are consistent with the styles, components and patterns in the GOV.UK Design System
+- production resources output the same markup as GOV.UK Frontend
+- it doesn't make it hard for teams to meet the [Service Standard](https://www.gov.uk/service-manual/service-standard)
```

Comment:
Can the last 2 bullets be made positive? Maybe something like:

- it allows teams to meet the Service Standard
- it meets the GOV.UK external linking policy